tapgarden/caretaker.go

```diff
@@ -411,26 +403,28 @@ func (b *BatchCaretaker) fundGenesisPsbt(ctx context.Context) (*FundedPsbt, erro
 	log.Infof("BatchCaretaker(%x): creating skeleton PSBT", b.batchKey[:])
 	log.Tracef("PSBT: %v", spew.Sdump(genesisPkt))
 
-	// If a fee rate was manually assigned for this batch, use that instead
-	// of a fee rate estimate.
 	var feeRate chainfee.SatPerKWeight
 	switch {
+	// If a fee rate was manually assigned for this batch, use that instead
+	// of a fee rate estimate.
 	case b.cfg.BatchFeeRate != nil:
 		feeRate = *b.cfg.BatchFeeRate
-		log.Infof("BatchCaretaker(%x): using manual fee rate",
-			b.batchKey[:])
+		log.Infof("BatchCaretaker(%x): using manual fee rate: %s, %d "+
```

Comment:
I think we should do `feeRate.FeePerKVByte() / 1000` instead of `feeRate.FeePerKVByte().FeeForVSize(1)` which is a bit confusing IMO.